src/platform/site-wide/mega-menu/tests/megaMenu.cypress.spec.js

```diff
@@ -64,6 +64,90 @@ const testMobileMenuSections = () => {
   cy.get('.vetnav-controller-open').contains('Menu');
 };
 
+const testMobileTabFocus = () => {
+  cy.get('[data-e2e-id="about-va-1"]').should('not.be.visible');
+  cy.get('#mega-menu-desktop #vetnav').should('not.exist');
+
+  cy.get('.vetnav-controller-open').contains('Menu');
+  cy.get('.vetnav-controller-open').click();
+  cy.get('.vetnav-controller-close').contains('Close');
+
+  // tabbing through first level menu wraps around to open/close button
+  cy.get('[data-e2e-id="mobile-home-nav-link"]')
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab();
+
+  cy.focused().should('have.attr', 'aria-controls', 'vetnav');
+
+  // shift tabbing through first level menu wraps around to open/close button
+  cy.get('[data-e2e-id="mobile-home-nav-link"]')
+    .tab({
+      shift: true,
+    })
+    .tab({
+      shift: true,
+    })
+    .tab({
+      shift: true,
+    })
+    .tab({
+      shift: true,
+    })
+    .tab({
+      shift: true,
+    })
+    .tab({
+      shift: true,
+    });
+
+  cy.focused().should('have.attr', 'aria-controls', 'vetnav');
+
+  // tabbing through second level menu wraps around to open/close button
+  cy.get('[data-e2e-id="mobile-home-nav-link"]')
+    .tab()
+    .click()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab()
+    .tab();
```

Comment:
If anyone knows a better way to do this, **please** let me know!

Reply:
Yeah, I'm in the same boat. I wonder if you can pass any params to the tab func which determine how many tabs to commit?

Reply:
I wonder if you could abuse cy.each() to do this? _Maybe_ cy.each() over an array of a given length? Probably not :( https://docs.cypress.io/api/commands/each#Syntax

Reply:
Also, `cy.tab()` is a thing? I don't see it in the docs.

Reply:
> Also, `cy.tab()` is a thing? I don't see it in the docs.

A plugin maybe?

Reply:
> A plugin maybe?

https://github.com/Bkucera/cypress-plugin-tab

Reply:
Maybe try this (I'm reading the README for the plugin):

`cy.get('[data-e2e-id="mobile-home-nav-link"]').tab()`
`cy.focused().click()`
`cy.focused().tab()`

Reply:
Oh I see. We've only been using that plugin for... _checks watch_ ...as long as we've been using Cypress. 🙄